app/renderer/src/components/Inspector/Source.jsx

```diff
@@ -20,27 +21,98 @@ const Source = (props) => {
     methodCallInProgress,
     mjpegScreenshotUrl,
     isSourceRefreshOn,
+    pageSourceSearchText,
     t,
   } = props;
 
+  /**
+   * Highlights the part of the node text in source tree that matches the search term.
+   */
+  const highlightNodeMatchingSearchTerm = (nodeText, searchText) => {
```

Comment:
it would be nice to add a unit test for this method

Reply:
Added necessaty unit tests.